src/pages/solution/places/behavior-reference.md

```diff
@@ -0,0 +1,146 @@
+
+#### Membership time to live
+
+**Membership time to live** specifies how long POI states (Current POI, Last Entered POI, Last Exited POI) remain valid. This duration applies globally to all POIs and has a default value of one hour. The duration is refreshed in the following cases:
+
+* On every POI entry or exit event.
+* When calling [`getNearbyPointsOfInterest`](/src/pages/solution/places/api-reference.md#getnearbypointsofinterest).
+
+To modify this value, use the `places.membershipttl` key. For additional details, see [Places configuration](/src/pages/solution/places/index.md#configuration-keys).
+
+## Places states
+
+There are three primary Places states: Current POI, Last Entered POI, and Last Exited POI. These states are also stored in device persistence and carry across app sessions. In a tags mobile property, rules can define Places conditions based on these states. Scenarios illustrating how these states change are provided in the [Scenarios](#scenarios) section.
+
+### Current POI
+
+When multiple POIs are simultaneously in the entered state, they are evaluated in the following order of precedence:
+
+1. Library rank: Lower rank values represent higher priority.  
+2. Radius: The POI with the smaller radius is prioritized.  
+3. Most recent entry signal or closest POI: The evaluation depends on the API used:  
+   * For [`processRegionEvent`](/src/pages/solution/places/api-reference.md#processregionevent), [`processGeofence`](/src/pages/solution/places/api-reference.md#processgeofence), and [`processGeofenceEvent`](/src/pages/solution/places/api-reference.md#processgeofenceevent), the POI associated with the most recent entry signal takes priority.  
+   * For [`getNearbyPointsOfInterest`](/src/pages/solution/places/api-reference.md#getnearbypointsofinterest), the POI closest to the provided latitude and longitude takes priority.  
+
+### Entered state
+
+A POI is in the entered state when an entry signal is sent using the [`processRegionEvent`](/src/pages/solution/places/api-reference.md#processregionevent) API on iOS or the [`processGeofence`](/src/pages/solution/places/api-reference.md#processgeofence) and [`processGeofenceEvent`](/src/pages/solution/places/api-reference.md#processgeofenceevent) APIs on Android.
+POIs remain in the entered state across app sessions until one of the following occurs:
+
+1. An exit signal is sent for the POI using the [`processRegionEvent`](/src/pages/solution/places/api-reference.md#processregionevent) API on iOS or the [`processGeofence`](/src/pages/solution/places/api-reference.md#processgeofence) and [`processGeofenceEvent`](/src/pages/solution/places/api-reference.md#processgeofenceevent) APIs on Android.  
+2. The membership time to live value expires.  
+3. The entered state is recalculated for all received POIs using the lat/lon provided when calling the [`getNearbyPointsOfInterest`](/src/pages/solution/places/api-reference.md#getnearbypointsofinterest) API.  
```

Comment:
let's check on this one too - i believe this is _a_ reason why the poi would exit an "entered" state, but i don't believe it is _always_ a reason why.  it may be a bit confusing to the reader.